contracts/src/Rollup.sol

```diff
@@ -115,6 +128,12 @@ contract Rollup is RollupBase {
         lastConfirmedAssertionID = _initialAssertionID;
         lastCreatedAssertionID = _initialAssertionID;
 
+        // Initialize role based access control
+        __AccessControlDefaultAdminRules_init(3 days, msg.sender);
```

Comment:
maybe move to `__RollupBase_init()` so we have a consistent place to initialize these contracts?